src/io/flutter/devtools/DevToolsManager.java

```diff
@@ -311,6 +351,31 @@ public void openBrowserAndConnect(String serviceProtocolUri, String page) {
       null
     );
   }
+
+  public void openPanel(String serviceProtocolUri, ContentManager contentManager, String tabName) {
+    String address = DevToolsUtils.generateDevToolsUrl(devtoolsHost, devtoolsPort, serviceProtocolUri, null) + "&embed=true&page=inspector";
```

Comment:
update DevToolsUtils to accept passing in embed=true and pass in the tabName instead of hardcoding to the inspector.

Reply:
`generateDevToolsUrl` has a `page` param but is this now inactive? e.g. the method can produce this URL (from a test): `"http://127.0.0.1:9100/?ide=Android-Studio&uri=http%3A%2F%2F127.0.0.1%3A50224%2FWTFTYus3IPU%3D%2F#timeline"` but adding `#timeline` doesn't seem to do anything when I try in devtools.